### DISCUSSION TITLE
Weekly Blog 7

### DISCUSSION THREAD

**Discussion by BenjaminB109** [SE-TINF22B6/Underwatch#81] [Announcements]
Hello everyone,
this time we would like to introduce you to the following:
1. Quality-Goals
2. an insight into how the map and various assets are designed and created.

# Quality-Goals
We choose the ISO/IEC 25010 standard as a template for our quality goals. If you are interested into learning more about this standard or even adopting it in you project, you can read everything about it on their website: https://iso25000.com/index.php/en/iso-25000-standards/iso-25010
We also have a mindmap which contains all the quality goals specified in the standard:
<img width="799" alt="Screenshot 2023-10-18 at 14 38 01" src="https://github.com/SE-TINF22B6/Underwatch/assets/87950162/a407dcaf-1f93-4ef1-a589-a6a199d75db9">
We have specified our main quality goals and some quality scenarios, which give a motivation for the mentioned quality goals. For an indepth look please refer to the Underwatch wiki. https://github.com/SE-TINF22B6/Underwatch/wiki/System-Architecture-Documentation#10-quality-requirements

We will take a look at the main quality goals of our project now.
| Goal                    | Description                                                      |
|-------------------------|------------------------------------------------------------------|
| 1. User Engagement      | The users should be engaged while playing the game               |
| 2. Self-descriptiveness | The game should be easy to understand with low efforts           |
| 3.  Installability      | The game should run on all major platforms (windows, linux, mac) |

These are the main quality goals. As you can see, the mainly revolve around usability and ease of use. These are core quality features of any fun game.
1. It should be easy to install
2. It should not be too complicated to learn, and players should not need to depend on external services to enjoy the game. For example, a wiki site on the internet, which forces players to leave the game.
3. The game should be engaging and fun.

# Game-Design
## Tiled
Let's start with the map:
to create this we use the Tiled program. This program is perfect for creating 2D maps. You can find a lot of free tilesets on the internet that you need to design the map. 

![grafik](https://github.com/SE-TINF22B6/Underwatch/assets/122724021/2c4a2c9e-684b-4506-8eeb-0a822e29b325)

On the left side you can see a tileset. There you can select the different textures to place them on the map. In the middle we have the map that you can build into the game with libgdx and on the right side you can see the different layers in which you can place the graphics. This is done to make it easier to address the different components in java.

## Pixelorama
We use Pixelorama to create weapons or ammunition, for example. This makes it easy to create pixel art. In the picture you can see the ammunition that you can collect later.

![grafik](https://github.com/SE-TINF22B6/Underwatch/assets/122724021/c0af34cf-e6b1-4210-b900-a53f5e37871e)

## Freesound
For the sound effects, we either record the sounds ourselves or download them from Freesound. All downloaded assets will of course be linked. On this page you can find a lot of different sounds in different versions, so we can choose the ones we like best.

![grafik](https://github.com/SE-TINF22B6/Underwatch/assets/122724021/23131aa0-02b4-4b97-8966-e9fa6180d9b1)

**Comment by Nahlam4** [comment]
Hey Underwatch team, once again a great Blog.

Also once again a slightly different standard to specify your quality goals. 
I think your chosen main goals are good, even though your third goal isn´t really quantifiable. 
 Nontheless, once your game is playable enough, I would gladly test your game and see if you have achieved them. 

I also like that you show the tools you are using to create your game assets.

Your DHBWhub team

**Comment by TadaXD20** [comment]
Hi Underwatch,

I love that you show us how your game comes together step by step. Your top 3 goals are well chosen. Because it is a game it is very wisely to of you to focus with two of three goals on the usability. The only thing I would like to see is why you chose exactly these three goals e.g. with an excerpt from your discussion (like the DHBWhub team did in their report).

I am looking forward to your next achievements ^^
Tanina from Team Plapy

**Comment by derrrobin** [comment]
Hi, I really like how you showed us how you created the design elements for your game. This helps us a lot to get an insight into the background of your project. You can talk about your design elements more often, as we as the Cookpit team find this very interesting. 

Your CookPit Team
